### PR TITLE
Upgrade govuk-frontend and the crown logo

### LINKED ISSUE #337: Update the crown
## What should change

govuk-frontend will soon be updating the crown in https://github.com/alphagov/govuk-frontend/issues/4019. For integrity of tech-docs-gem sites, as official sources, I believe it is important to pull in this change, once it has been released.

### PR DESCRIPTION
**Pull request by kr8n3r**
Closes  #337 #343

## What’s changed

- Update govuk-frontend from v4.4.1 to v4.7
- Update crown SVG and favicon to Tudor crown
- Manually adjust SVG crown with styles from govuk-frontend v4.8
- Suppress IE8 mixin deprecation warning introduced in govuk-frontend v4.6

Ideally we would upgrade to govuk-frontend v4.8 and not have to bring in the minor manual SVG alignment, but as govuk-frontend v4.8 introduced SCSS source maps, the tech-docs gem compiler includes them but not any styles.

## Identifying a user need

We need to update the crown before March 1

## How to test with your tech docs

To see how your changes to the gem affect your website, point your website's Gemfile to your local checkout of this branch:

```rb
gem 'govuk_tech_docs', path: '../tech-docs-gem'
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "lint": "standard"
   },
   "dependencies": {
-    "govuk-frontend": "^4.4.1"
+    "govuk-frontend": "4.7.0"
```
Comment (richardTowers) [inline package.json:9]: Is it worth specifying a range? I'm not sure if that makes any difference in practice...

**Review by richardTowers — APPROVED** [review]
Code looks good to me, although I haven't done any testing. One trivial comment, but otherwise 👍🏻

File: package.json (hunk 1 above)
Comment (kr8n3r) [reply package.json:9]: thanks Richard, it's highly unlikely that 4.7.x will be release, but for peace of mind I can do ~

**Review by huwd — APPROVED** [review]
Tested on two internal GDS docs sites, both worked. Thanks Jani!

**Review by abutters — APPROVED** [review]
Tested locally on [customs-declarations-end-to-end-service-guide](https://github.com/hmrc/customs-declarations-end-to-end-service-guide). Works well. Thank you very much!

**Pull request merged**